dateparser/conf.py

```diff
@@ -31,19 +31,21 @@ class Settings:
     _pyfile_data = None
     _mod_settings = dict()
 
-    def __init__(self, settings=None):
-        if settings:
-            self._updateall(settings.items())
-        else:
+    def __init__(self, **kwargs):
```

Comment:
Why the switch to `**kwargs`?

Reply:
To pass other keyword arguments to Settings class which is required to get different `registry_key` for settings with different locale/language arguments.